src/Accounts/Authentication/Sanitizer/Services/DefaultSanitizerService.cs

```diff
@@ -12,13 +12,19 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Security.Utilities;
+
 using System.Collections.Generic;
-using System.Text.RegularExpressions;
 
 namespace Microsoft.Azure.Commands.Common.Authentication.Sanitizer.Services
 {
+
     internal class DefaultSanitizerService : ISanitizerService
     {
+        private SecretMasker secretMasker =
+            new SecretMasker(WellKnownRegexPatterns.HighConfidenceMicrosoftSecurityModels,
+                             generateSha256Hashes: true);
+
         public string SanitizedValue => "******";
```

Comment:
>"******"; [](http://example.com/codeflow?start=40&length=9)

Vincent, currently this argument is not honored by the OSS masker, which currently injects `+++` for every redaction find. The next update of the library will actually inject a hash of the redacted find that can be used to correlate the redacted secret with other telemetry logs. We probably want this option to be configurable and we probably want the library to be configurable to update the default sanitized value. I'll look into this for 1.4.4.